### PR TITLE
Feature/rqm update changes

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [x] Sanity Tests are successfully executed for Existing Functionality
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [x] Resolve code review comments

### CODE REVIEW

**Comment by prashelke** [comment]
Unit test is added in Rqm Repository we are not able to added unit test in ginger because we only used Rqm Repository dll in ginger so need to add into Regression suite

File: Ginger/Ginger/App.config [hunk 1]
```diff
@@ -1,170 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (Maheshkale447) [inline Ginger/Ginger/App.config:1]: App.config not needed anymore?

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/App.config (hunk 1 above)
Comment (prashelke) [reply Ginger/Ginger/App.config:1]: we need it I have updated it

Comment (prashelke) [reply Ginger/Ginger/App.config:1]: we need it

**Pull request merged**